docs/user/filesystem/partition_schemes.rst

```diff
@@ -0,0 +1,128 @@
+.. _fsds_partition_schemes:
+
+Partition Schemes
+=================
+
+Partition Schemes can be defined using a JSON/typesafe configuration or by a common name. By default these schemes all
+utilize "leaf storage" with a one up sequence number. This allows for appending data when it arrives out of order.
+
+Common Date schemes are:
+
+* minute
+* hourly
+* daily
+* monthly
+* julian-minute
+* julian-hourly
+* julian-daily
+
+Common geometry schemes are:
+
+* z2-2bit
+* z2-4bit
+
+Common combined schemes are created by using a comma. For example:
+
+* hourly,z2-2bit
+* julian-daily,z2-2bit
+
+Leaf vs Bucket Storage
+----------------------
+
+As previously mentioned, the default storage mode is leaf storage. Leaf storage means that the last component of the
+partition scheme is the prefix of the parquet file name. A sequence number is then appended to the filename before the
+file extension. For example, a partition scheme of ``yyyy/MM/dd`` would produce the storage path
+``2016/01/01_0000.parquet`` for the first file created for the partition ``2016/01/01``. The next file (created by a
+new ingest process) for the same partition would result in a file named ``2016/01/01_0001.parquet`` and then
+``2016/01/01_0002.parquet`` and so on.
+
+By contrast, bucket storage uses a directory for the final component of the scheme. For our partition example above
+the storage paths would be ``2016/01/01/0001.parquet``, ``2016/01/01/0002.parquet``, and ``2016/01/01/0003.parquet``.
```

Comment:
Did you have a chance to work out the "_" as a separator?  (Rather than slashes?)